extensions/standard-processors/tests/unit/HashContentTest.cpp

```diff
@@ -147,17 +147,30 @@ TEST_CASE("TestingFailOnEmptyProperty", "[HashContentPropertiesCheck]") {
                                                                      core::Relationship("success", "description"), true);
   plan->setProperty(md5processor, org::apache::nifi::minifi::processors::HashContent::HashAttribute.getName(), MD5_ATTR);
   plan->setProperty(md5processor, org::apache::nifi::minifi::processors::HashContent::HashAlgorithm.getName(), "MD5");
-  plan->setProperty(md5processor, org::apache::nifi::minifi::processors::HashContent::FailOnEmpty.getName(), "true");
+
+  using processors::HashContent;
```

Comment:
we could move this `using` to the beginning of the test case and get rid of all the fully qualified `HashContent` references in this test